README.md

```diff
@@ -510,6 +511,40 @@ data:
 You're welcome to raise issues related to feature requests, bugs, or give us general feedback on this project's GitHub Issues page. 
 The SAP BTP service operator project maintainers will respond to the best of their abilities. 
 
+[Back to top](#sap-business-technology-platform-sap-btp-service-operator-for-kubernetes)
+
+## Deleting the Operator
+
+To delete the operator, run the following command:
+`helm uninstall <release name> -n <name space>`
+
+Example: 
+
+ >   ```
+  >   helm uninstall sap-btp-operator -n sap-btp-operator
+
+
+**Note:** All service bindings and service instances are also deleted during this process and the expected time out is up to 5 minutes.
```

Comment:
I would remove "and the expected time out is up to 5 minutes", it is written is the possible issues section.

Reply:
Done